app/policies/feedback_policy.rb

```diff
@@ -0,0 +1,17 @@
+class FeedbackPolicy < ApplicationPolicy
+  def destroy?
+    if user.is_a?(User)
```

Comment:
```suggestion
    if user.present?
```
Ça se simplifie sans doute un peu si on déplace `creator?` ici. Tu en penses quoi?

Reply:
ici aussi on veut faire deux choses différentes si c'est un User ou un Expert